src/olympia/devhub/utils.py

```diff
@@ -256,6 +259,9 @@ def __init__(self, file_, addon=None, listed=None, final_task=None):
                 tasks.forward_linter_results.s(file_.pk),
             )
 
+            if waffle.switch_is_active('enable-yara'):
+                tasks_in_parallel = (*tasks_in_parallel, run_yara.s(file_.pk))
```

Comment:
Nit: any reason for `tasks_in_parallel` to be a `tuple` and not a `list` we'd just `append()` task signatures to ?

Reply:
No, it is mostly because the celery example you started last week used a tuple. I think both are valid.